extra/sponge.importorder

```diff
@@ -0,0 +1,7 @@
+#Organize Import Order
```

Comment:
Ah. Well, we don't have this in our IntelliJ configuration, and I don't like adding only the most common packages, it should work correctly for any import. For now I would just do it like our IntelliJ configuration.